### DISCUSSION TITLE
請問哪裡可以關閉自動翻譯的功能？

### DISCUSSION THREAD

**Discussion by imtaku** [immersive-translate/immersive-translate#2304] [Help]
### 概述 | Summary

請問哪裡可以關閉自動翻譯的功能？我到處都找不到，可以的話，請提供截圖，謝謝

**Comment by keyboard3** [comment]
检查下你的面板
![QQ_1732525753431](https://github.com/user-attachments/assets/0af3979e-f6f1-4c58-855d-8366cac62771)
